### DISCUSSION TITLE
Method Filament\Panel::getDomains does not exist.

### DISCUSSION THREAD

**Discussion by scoba6** [filamentphp/filament#8054] [General]
Hi Everyone,

I get this error (Method Filament\Panel::getDomains does not exist.) after update Filament to V3 (Livewire now stable). Any solution?

Thanks @danharrin

**Comment by ehs4nm** [comment]
Hi there 

I have the same error:

In Macroable.php line 72:                                                           
        Method Filament\Panel::getDomains does not exist.
